content/docs/12-enterprise-version/03.5-helm-chart-install-reference.md

```diff
@@ -0,0 +1,260 @@
+---
+title: "Helm Chart Install Reference"
+metaTitle: "Helm Chart Install References"
+metaDescription: "Reference page for Pallete Helm chart self-installation values.yaml file parameters."
+icon: ""
+hideToC: False
+fullWidth: False
+---
+
+# Helm Chart Install Reference 
+
+You can use the Palette Helm Chart to install Palette in a multi-node Kubernetes cluster in your production environment. The Helm chart allows you to customize values in the values.yaml file. This reference guide lists and describes parameters available in the values.yaml file from the Helm Chart for your installation. To learn how to install Palette using the Helm Chart, refer to [Helm Chart Mode](https://docs.spectrocloud.com/enterprise-version/deploying-palette-with-helm).  
+
+# Required parameters
+
+The following parameters are required:
+
+- **env.rootDomain**- Used to configure the domain for the Palette installation. **Note:** You should create a DNS record separately, and it should be a wildcard to account for Organization prefixes. Review the [environment parameters](#environment) to learn more. 
+
+- **natsUrl**- The URL format specifies configuring the NATS server(s) to the IP address and port. Review the [Network Address Translation (NATS) parameters](#network-address-translationnats) to learn more. 
+ 
+- Refer to the [Registry and Palette Artifact Repository parameters](#registry-and-palette-artifact-repositorypar) used to specify the Docker registry where the chart images are stored and the Spectro Cloud Admission Registrar (SCAR), respectively.
```

Comment:
This paragraph should start with the name of the parameter or section. It's a bit awkward in it's current form. Also, I noticed you are using PAR in the doc, so we should remove SCAR and replace with PAR.